src/jit/unwindamd64.cpp

```diff
@@ -121,7 +121,15 @@ void Compiler::unwindPush(regNumber reg)
     unsigned int cbProlog = unwindGetCurrentOffset(func);
     noway_assert((BYTE)cbProlog == cbProlog);
     code->CodeOffset = (BYTE)cbProlog;
-    if (RBM_CALLEE_SAVED & genRegMask(reg))
+    if (RBM_CALLEE_SAVED & genRegMask(reg)
```

Comment:
Please add another set of parens around this expression, so we don't have to remember operator precedence between & and ||